src/Perpetuum/Services/EventServices/EventProcessors/DayTimeEventProcessor.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Perpetuum.ExportedTypes;
+using Perpetuum.Services.Daytime;
+using Perpetuum.Services.EventServices.EventMessages;
+using Perpetuum.Zones;
+using Perpetuum.Zones.Effects.ZoneEffects;
+
+namespace Perpetuum.Services.EventServices.EventProcessors
+{
+    public class DayTimeEventProcessor : EventProcessor<EventMessage>
+    {
+        private IZone _zone;
+        private readonly Lazy<ZoneEffect> _dayEffect;
+        private readonly Lazy<ZoneEffect> _nightEffect;
+        public DayTimeEventProcessor(IZone zone)
+        {
+            _zone = zone;
+            _dayEffect = new Lazy<ZoneEffect>(GetDayEffect);
+            _nightEffect = new Lazy<ZoneEffect>(GetNightEffect);
+        }
+
+        private ZoneEffect GetDayEffect()
+        {
+            return new ZoneEffect(_zone.Id, EffectType.effect_daytime_day, true);
+        }
+
+        private ZoneEffect GetNightEffect()
+        {
+            return new ZoneEffect(_zone.Id, EffectType.effect_daytime_night, true);
+        }
+
+        private static readonly int _sunrise = 150;
+        private static readonly int _sunset = 750;
+
+        private bool IsDay(DayTimeMessage msg)
```

Comment:
This is a little strange to read, `DayTimeEventProcessor.IsDay(...)`. Isn't it day? It's right in the name... As opposed to `TimeOfDayEventProcessor.IsDay(...)`.

Reply:
Agree that the class names could use clarification, but the method name is fine.